src/turbopuffer/types/custom.py

```diff
@@ -1,8 +1,10 @@
 # Code generated by turbopuffer-apigen. DO NOT EDIT.
 
-from typing import Any, Tuple, Union, Literal, Sequence
+from typing import Any, Tuple, Union, Literal, Sequence, TypedDict
 
 AggregateBy = Tuple[Literal["Count"], str]
+ExprRefNew = TypedDict("ExprRefNew", {"$ref_new": str})
+Expr = ExprRefNew
```

Comment:
nit: any reason to order Expr below ExprRefNew? It's reversed in the other languages.

Reply:
(Discussed live, but for posterity: Python doesn't support forward references.)